app/services/resources/resource_evaluation_service.rb

```diff
@@ -0,0 +1,13 @@
+module Resources
+  class ResourceEvaluationService
```

Comment:
ahi pasaria el `resource_id` al  momento de instanciar el servicio`def initialize(resource_id)`
onda → `Resources::ResourceEvaluationService.new(resource_id:).calculate_average_evaluation`


lo haria asi, avisame si quieres que te lo comento de vivo:


```
module Resources
  class ResourceEvaluationService
    def initialize(resource_id)
      @resource_id = resource_id
    end

    def calculate_average_evaluation
      average_evaluation.presence || 'Sin evaluación'
    end

    private

    def rounded_evaluation
      average_evaluation&.round(1)
    end

    def average_evaluation
      @average_evaluation ||= ResourceEvaluation.where(resource_id: @resource_id)
                                                .average(:evaluation)
    end
  end
end
```

y ahi puedes tener en el servicio un e

Reply:
Done :) Thanks for all the comments!